src/help/faq/README.md

```diff
@@ -25,7 +25,14 @@ If you're having problems with the app, please check here before posting. If you
 ---
 
 ### App crashes on launch
-If you are using the **Public build**, check if you are on iOS 13.4+ or iPadOS 13.4+. If you are using the **Beta / Patron build**, ping @Paper on the Discord #supporter channel.
+If you are using the **Public build**, check if you are on iOS 13.4+ or iPadOS 13.4+. If you are using the **Beta / Patron build**, ping @Paper on the Discord #supporter-chat channel.
+
+---
+
+### Both Paperback and AltStore crash on launch
+Applications installed with AltStore need to be refreshed every seven days. While AltServer and Mail or iCloud/iTune are open and your device is connected to the same network as your computer, AltStore will try to refresh the app automatically.
+If Paperback and AltStore both crash on launch it is likely that their certificate is expired.
+Without uninstalling anything connect your iPhone/iPad to your computer. Open AltServer and Mail or iCloud/iTune then select "**Install AltStore**" from AltServer menu.
```

Comment:
I dunno if it's a regional thing, but I've never seen it called `iTune`, it's always been `iTunes`. Your call.

```suggestion
Without uninstalling anything connect your iPhone/iPad to your computer. Open AltServer and Mail or iCloud/iTunes then select "**Install AltStore**" from AltServer menu.
```